src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+export const addBook = (book) => ({
+  type: 'bookstore/books/ADD_BOOK',
```

Comment:
- [ ] **Optional:** Your actions are conventionally saved as variables at the top of your **reducer** files. E.g

`const ADD_BOOK = 'bookstore/books/ADD_BOOK', `

This way you avoid typos when used multiple times through your codebase most especially in bigger projects.